.gitignore

```diff
@@ -1,3 +1,9 @@
+# Exlclude build Output
+dist/
+
+
+######### App / OS-specific guff ##########
+
 ### macOS ###
 *.DS_Store
```

Comment:
Good spot.

This was auto-generated by https://www.gitignore.io/ . Weirdly, they don't have the `*` on other macOS ignores, so it looks deliberate. However, I can't find any information suggesting that the file is ever called anything other than just `.DS_Store`.